{"id":13969,"className":"java.lang.IllegalArgumentException","methodName":"IllegalArgumentException","signature":"\u003cjava.lang.IllegalArgumentException: void IllegalArgumentException(java.lang.Throwable)\u003e","javadoc":"/** \n * Constructs a new exception with the specified cause and a detail message of \u003ctt\u003e(cause\u003d\u003dnull ? null : cause.toString())\u003c/tt\u003e (which typically contains the class and detail message of \u003ctt\u003ecause\u003c/tt\u003e). This constructor is useful for exceptions that are little more than wrappers for other throwables (for example,  {@link java.security.PrivilegedActionException}).\n * @param cause the cause (which is saved for later retrieval by the{@link Throwable#getCause()} method).  (A \u003ctt\u003enull\u003c/tt\u003e value ispermitted, and indicates that the cause is nonexistent or unknown.)\n * @since  1.5\n */\n","tf":{"freq":{"littl":1,"detail":2,"save":1,"link":2,"typic":1,"use":1,"specifi":1,"indic":1,"wrapper":1,"action":1,"valu":1,"privileg":1,"constructor":1,"argument":1,"except":4,"nonexist":1,"messag":2,"later":1,"unknown":1,"illeg":1,"ispermit":1,"caus":8,"secur":1,"retriev":1,"contain":1,"string":1,"exampl":1,"method":1,"java":1,"construct":1,"throwabl":2},"maxFreq":8,"totalTerms":45},"loc":3}